gadget/ondisk.go

```diff
@@ -261,3 +261,30 @@ func MockSysfsPathForBlockDevice(f func(device string) (string, error)) (restore
 	sysfsPathForBlockDevice = f
 	return restore
 }
+
+// Build a map of gadget yaml index to OnDiskStructure by assuming that the
```

Comment:
This comment should start with the function name

Reply:
Done